frontend/src/components/Alerts/AlertsBanner.tsx

```diff
@@ -21,15 +21,27 @@ const Center = styled.div`
     align-items: center;
 `
 
+export enum AlertCategory {
+    ERROR,
+    WARNING,
+    SUCCESS,
+}
+
 interface AlertProps {
     children: ReactNode
     dismissAlert: () => void
+    alertCategory: AlertCategory
 }
 
-export function AlertBanner({ children, dismissAlert }: AlertProps) {
+export function AlertBanner({ children, dismissAlert, alertCategory }: AlertProps) {
+    let bannerColor = tokens.colors.interactive.danger__resting.hex
+
+    if (alertCategory === AlertCategory.WARNING) bannerColor = tokens.colors.interactive.warning__resting.hex
+    if (alertCategory === AlertCategory.SUCCESS) bannerColor = tokens.colors.interactive.success__resting.hex
+
     return (
         <>
-            <StyledCard variant="danger" style={{ boxShadow: tokens.elevation.raised }}>
+            <StyledCard variant="danger" style={{ boxShadow: tokens.elevation.raised, backgroundColor: bannerColor }}>
```

Comment:
Seems redundant with both variant and bannerColor

Reply:
I prefer using variant="info"/"warning"/"danger". Currently bannercolor is very strong which makes it harder to read. Also the failed icon is the exact same color and disappear 
![image](https://github.com/equinor/flotilla/assets/33518988/9072d8c6-98ff-4636-994b-4bc2e929af43)

Reply:
The current "string" colors ok as long as we change the color of the content in the existing banners such that the icons and the background have different colors.

Suggest consulting with UX designer about colors. Don't think it's necessary to change from card to banner